src/content/RepoPage/RepoPage.js

```diff
@@ -101,12 +101,12 @@ const RepoPage = () => {
         <div className="bx--col-lg-16">
           <Query query={REPO_QUERY}>
             {({ loading, error, data: { organization } }) => {
-              // Wait for the request to complete
+              // Waiting for the request to complete
               if (loading)
                 return (
                   <DataTableSkeleton
                     columnCount={headers.length + 1}
-                    rowCount={10}
+                    rowCount={12}
```

Comment:
Why is it 12, not 10? 😄

Reply:
I think it was overwritten by the merge commit :'D